composer.json

```diff
@@ -82,8 +82,7 @@
         "phpunit/phpunit": "^9.3.11",
         "symfony/phpunit-bridge": "^4.4.14 || ^5.1.6",
         "symfony/yaml": "^5.0",
-        "thecodingmachine/phpstan-safe-rule": "^1.0",
-        "vimeo/psalm": "^4.0"
```

Comment:
psalm uses `composer/package-versions-deprecated` https://github.com/vimeo/psalm/blob/4.1.1/composer.json#L26 which has an issues with Box https://github.com/box-project/box/issues/460

that is why I decided to move it out from composer dependencies. 
Unfortunately we can not add `composer/package-versions-deprecated` to the `confict` section of composer.json as it may cause some issues for those users who prefer to use Infection as composer dependency.